server/i18n/en.json

```diff
@@ -4850,6 +4850,10 @@
     "id": "app.bot.permenent_delete.bad_id",
     "translation": "Unable to delete the bot."
   },
+  {
+    "id": "app.channel.add_member.deleted_user.app_error",
+    "translation": "Unable to add a deactivated user as a member of the channel."
```

Comment:
Should this error be less descriptive? Information disclosure... I know its a valid user id of a deactivated user.

Reply:
Nice! Well you tell me, I'm not really sure what else to use instead.